### PR TITLE
Checks adjustments

### PR DESCRIPTION
**Pull request by anthonyshew**
- Removes precommit.
- Adjust linting and formatting commands.

### CODE REVIEW

**Review by PickleNik — APPROVED** [review]
GLTM

File: apps/admin/package.json [hunk 1]
```diff
@@ -7,8 +7,10 @@
     "build": "next build",
     "clean": "rm -rf .next dist node_modules",
     "dev": "dotenv -e ../../.env next dev -- -p 3001",
-    "format": "prettier -w \"src/**/*.{ts,tsx,css,cjs,mjs,json}\"",
-    "lint": "eslint .",
+    "format": "prettier . --check --cache --cache-location='node_modules/.cache/prettiercache'",
```
Comment (jhechtf) [inline apps/admin/package.json:10]: IDK if this is just me, but this creates a new directory in a bunch of places that is named `'node_modules/.cache/prettiercache'`. I believe the intent here is to have the cache stored in the `node_modules/.cache/prettiercache`, which might make more sense to change this to 

```suggestion
"format": "prettier . --check --cache --cache-location=\"node_modules/.cache/prettiercache\"",
```

Comment (anthonyshew) [reply apps/admin/package.json:10]: Are you on Windows? I wasn't aware of that, will change.

File: .eslintrc.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 module.exports = {
   root: true,
   extends: [require.resolve('./tooling/config-eslint/node')],
+  ignorePatterns: ['apps/**', 'packages/**', 'tooling/**'],
```
Comment (jhechtf) [inline .eslintrc.js:5]: Correct me if I'm wrong but doesn't this remove linting from the majority of the repo?

Comment (anthonyshew) [reply .eslintrc.js:5]: This file only gets used by the root of the repository. In workspaces, you'll find `.eslintrc.js` files that have `root: true` so they won't resolve any rules higher than there.

I do seem to have a misconfiguration elsewhere, though. 😄

Comment (anthonyshew) [reply .eslintrc.js:5]: Ah, nope, I'm all good. Forgot that we have a lot of rules turned off right now in `tooling/config-eslint/utils/rules.js`.

If you want to see the linter get more strict, you can comment out some rules in there and run `turbo lint --force` and you'll be given issues.

**Review by jhechtf — APPROVED** [review]
LGTM

**Comment by Hacksore** [comment]
> GLTM

GOOD LOOKING TO ME

**Pull request merged**